src/sinks/splunk_hec/metrics/config.rs

```diff
@@ -57,18 +72,29 @@ pub struct HecMetricsSinkConfig {
     /// The name of the index where to send the events to.
     ///
     /// If not specified, the default index is used.
+    #[configurable(metadata(docs::examples = "{{ host }}", docs::examples = "custom_index"))]
     pub index: Option<Template>,
```

Comment:
same question as in the other PR about default

Reply:
Same as [other PR](https://github.com/vectordotdev/vector/pull/16334#discussion_r1101251795).